docs/book/src/topics/custom-images.md

```diff
@@ -217,6 +217,96 @@ This will make API calls to create Virtual Machines or Virtual Machine Scale Set
 
 In the case of a third party image, you must accept the license terms with the [Azure CLI][azure-cli] before consuming it.
 
+## Example: CAPZ with Mariner Linux
+
+To clarify how to use a custom image, let's look at an example of using [Mariner Linux][mariner] with CAPZ.
```

Comment:
I see the official docs refer to it as "CBL-Mariner" or "CBL-Mariner Linux" we use the same terminology everywhere in this doc?

Reply:
Ok, I am convinced :)